### PR TITLE
disable MoveTo::reverse()

### PR DESCRIPTION
**Pull request by fusijie**

### CODE REVIEW

**Comment by fusijie** [comment]
For the issue: https://github.com/cocos2d/cocos2d-x/issues/10589

File: cocos/2d/CCActionInterval.h [hunk 1]
```diff
@@ -549,6 +549,7 @@ class CC_DLL MoveTo : public MoveBy
     // Overrides
     //
     virtual MoveTo* clone() const override;
+    virtual MoveTo* reverse(void) const  override;
```
Comment (minggo) [inline cocos/2d/CCActionInterval.h]: Don't need void.

**Comment by minggo** [comment]
[ci rebuild]

**Comment by fusijie** [comment]
OK, I have removed the `void`.

**Pull request merged**